Sources/SPMBuildCore/BuildParameters.swift

```diff
@@ -240,7 +240,11 @@ public struct BuildParameters: Encodable {
         case .library(.static):
             return RelativePath("lib\(product.name)\(triple.staticLibraryExtension)")
         case .library(.dynamic):
-            return RelativePath("lib\(product.name)\(triple.dynamicLibraryExtension)")
+            if self.triple.os == .windows {
+              return RelativePath("\(product.name)\(triple.dynamicLibraryExtension)")
+            } else {
+              return RelativePath("lib\(product.name)\(triple.dynamicLibraryExtension)")
+            }
```

Comment:
we can probably make this into a case in the switch which would read better imo

Reply:
I care about readability, so I'll change that in a follow up.

Reply:
#2865